test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -199,3 +228,45 @@ func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int,
 	err = e2epod.WaitForPodSuccessInNamespaceTimeout(ctx, f.ClientSet, pod.ObjectMeta.Name, f.Namespace.Name, 300*time.Second)
 	gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(ctx, f, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name))
 }
+
+func injectError(ctx context.Context, f *framework.Framework) {
+	yes := true
+	podSpec := &v1.Pod{
```

Comment:
you will need to ensure the pod runs on the node that has QAT. perhaps using a `Job` is a better fit since the goal is to just run to completion. In general, #1582 e2e changes can give you ideas.

Reply:
Doesn't `JustBeforeEach` do that work?